experiments/list_unsupported.sh

```diff
@@ -34,8 +34,30 @@ done
 
 echo "$1: Unsupported features\n" > "$file_name".txt
 
+export ADA_INCLUDE_PATH=/opt/gnat/lib/gcc/x86_64-pc-linux-gnu/4.9.4/rts-native/adainclude/
+if [ ! -d "$ADA_INCLUDE_PATH" ]; then
+   echo >&2 "Ada include path does not exists!"
+   exit 4
+fi
+
+PATH=/opt/gnat/bin:${PATH}
+if ! command -v gnat > /dev/null; then
+   echo >&2 "Gnat not on PATH!"
+   exit 5
+fi
+
+GPR_PROJECT_PATH=/opt/gnat/lib/gnat
+if [ ! -d "$GPR_PROJECT_PATH" ]; then
+   echo >&2 "GPR project path does not exists!"
+   exit 6
+fi
+
 for filename in $(find ${path} -name *.adb); do
    gnat2goto ${include_path} "${filename}" >>"$file_name".txt 2>&1
+   if [ `echo $?` -gt 0 ]; then
+      echo >&2 "Errors occurred during feature collection."
```

Comment:
Assuming gant2goto failed with a non-zero exit code, does the log of what failed get written anywhere? If so, it's worth the error message saying something like "see /foo/blah/wibble/log.txt for details"

Reply:
Done.